cli/tests/data/package/create-directory/marathon.json.mustache

```diff
@@ -5,8 +5,8 @@
   "cpus": {{cassandra.framework.cpus}},
   "mem": {{cassandra.framework.mem}},
   "uris": [
-    "{{package.assets.uris.cassandra-mesos.tar.gz}}",
-    "{{package.assets.uris.jre-linux.tar.gz}}"
+    "{{resource.assets.uris.cassandra-mesos}}",
+    "{{resource.assets.uris.jre-linux}}"
```

Comment:
@tamarrow This is a feature of the bundle command so we should document this behaviour here. The Universe specification doesn't say anything about making zip packages.

Also, I am not sure I follow your comment around `package import`. The mustache templates should already be using `-` instead of `.` or else `version-2.x` branch in the `universe` repo wouldn't work. During the import operation we don't touch the name of the properties in `resource.json`. We only `rebase` the URLs in their values.